vochain/offchaindatahandler/offchaindatahandler.go

```diff
@@ -149,7 +149,12 @@ func (c *OffChainDataHandler) OnRevealKeys(pid []byte, priv string, txindex int3
 func (c *OffChainDataHandler) OnProcessStatusChange(pid []byte,
 	status models.ProcessStatus, txindex int32) {
 }
-
+func (c *OffChainDataHandler) OnSetAccount(addr []byte, account *vochain.Account) error {
```

Comment:
AFAIK the address is found inside the account

Reply:
it is not, the address have never been part of the account struct

Reply:
You'r right, the address is the key for the merkle tree storing the Account values.